tests/plugins/ml/train_predict/predict.yaml

```diff
@@ -0,0 +1,74 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test the prediction of new data with trained model.
+distributions:
+  excluded:
```

Comment:
Do you know for sure that these APIs don't exist in these environments? Otherwise remove the `excluded` parts.